tfs/src/main/java/hudson/plugins/tfs/telemetry/TelemetryContextInitializer.java

```diff
@@ -111,7 +111,8 @@ private String getUserId() {
         final String userName = getSystemProperty(SYS_PROP_USER_NAME);
         final String fakeUserId = MessageFormat.format("{0}@{1}", userName, computerName);
 
-        return DigestUtils.sha1Hex(fakeUserId);
+        //FIXME: was sha1Hex, but this is available only in commons codec 1.7. TFS SDK 14.0.3 bundles older version
```

Comment:
I don't think this needs to be fixed (i.e. you can remove the comment). The idea here is to keep the user id unique to the telemetry server, but not allow anyone to actually identify the user. So, as long as the user id is obfuscated, there shouldn't be a problem.